documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -104,7 +104,9 @@ on GitHub.
 * `TestInstancePostProcessors` registered on test methods are now invoked.
 * There are two new signatures for `Assertions.fail`: `Assertions.fail(Throwable cause)` and
   `Assertions.fail(String message, Throwable cause)`.
-
+* New `Assertions.assertLinesMatch()` comparing lists of `String` featuring `Object::equals` and
+  regular expression checks. It also provides a fast-forward mechanism to skip actual lines that
+  are expected to change in each invocation, e.g. duration, timestamps and stack traces.
```

Comment:
@sormuras I know at a basic level what you're trying to say here, as I've looked at the Javadoc and tests for this new assertion, but I don't think newcomers would understand from this rather sparse description what the assertion does or what its purpose is.

How about something like this?

> Introduced a new `Assertions.assertLinesMatch()` assertion, which is similar to `Assertions.assertIterableEquals()`, except that it executes on `List<String>`s rather than `Iterable<?>`s, and the expected lines can be a combination of literal lines to match against; regular expressions; and representations of a new concept called "fast-forward", which allows some of the actual lines to be exempted from checking. See the Javadoc for more details.